### ISSUE TITLE
Noto Kufi Arabic doesn't support HEH GOAL properly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [googlei18n/noto-fonts#34]
```
It shows it as its final form when it should be in initial form.

The character is important for Urdu.
```

Original issue reported on code.google.com by `roozbeh@google.com` on 15 Jul 2014 at 7:15

**Comment by behdad** [comment]
cc @roozbehp @kmansourMT

**Comment by kmansourMT** [comment]
This error has been fixed in the v1.04 release.
 
![hehheh-red](https://cloud.githubusercontent.com/assets/12738181/11596046/a8d37700-9a67-11e5-9226-a40f4e1118d6.png)
